project_timesheet_time_control/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa
+# © 2016 Pedro M. Baeza
+# © 2016 Sergio Teruel
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Project timesheet time control',
+    'version': '9.0.1.0.0',
+    'category': 'Project',
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
+              'Antiun Ingeniería S.L.',
```

Comment:
Yeah, we haven't still touched this. Even the credits are outdated :wink:

I'll do it this afternoon